sherlock/sherlock.py

```diff
@@ -554,19 +570,7 @@ def main():
     args = parser.parse_args()
 
     # Check for newer version of Sherlock. If it exists, let the user know about it
```

Comment:
Doesn't seem like a bad change to me, but could you please move the previous comment into a doc string?

Reply:
l have just added the doc string!